res/css/photon/radio-button.css

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 .photon-radio {
-  border: none;
+  border: 1px solid var(--grey-90-a30);
```

Comment:
Because I looked at this style to define the other one, I realized that it wasn't accessible like it was (it was too pale compared to the background)... and then I've seen that the spec in https://design.firefox.com/photon/components/radio-buttons.html actually wants a border, so here it is :)